iphone/cli/hooks/package.js

```diff
@@ -150,7 +125,9 @@ exports.init = function (logger, config, cli) {
 
 					// move the finished archive directory into the correct location
 					fs.existsSync(archivesDir) || wrench.mkdirSyncRecursive(archivesDir);
-					fs.renameSync(stagingArchiveDir, dest);
+					appc.fs.copyDirSyncRecursive(stagingArchiveDir, dest, {
```

Comment:
Node does not have any facilities to copy a directory. Wrench does, but Wrench doesn't do logging of which files are being copied. So, I added a `copyDirSyncRecursive()` to node-appc which does the copying and logs which files are being copied.